testutils/helper/install.go

```diff
@@ -155,8 +155,9 @@ func (h *SoloTestHelper) InstallGloo(ctx context.Context, deploymentType string,
 	for _, opt := range options {
 		opt(io)
 	}
-	if err := exec.RunCommand(h.RootDir, io.Verbose, io.GlooctlCommand...); err != nil {
-		return errors.Wrapf(err, "error while installing gloo")
+
+	if err := glooctlInstallWithTimeout(h.RootDir, io, time.Minute*2); err != nil {
```

Comment:
I picked 2 minutes here rather than using the passed-in timeout, as the passed timeout is more for "did the testrunner set up all of gloo within this time limit". For `glooctl install` to return however, just the certgen pod needs to start, which involves pulling a single ~25MB docker image from quay. In practice this should generally return in under a second on systems with this image cached, or in < 5 seconds on any system with a fast internet connection (ie our CI pipelines).

2 Minutes felt like a decent buffer to account for a miss or two if Quay is having a bad day, without waiting forever for failing the test if something really is wrong. Open to opinions though if others feel differently.